README.md

```diff
@@ -119,6 +119,7 @@ consul-token           |                       | The Consul ACL token
 consul-tag             | `marathon`            | Common tag name added to every service registered in Consul, should be unique for every Marathon-cluster connected to Consul
 consul-timeout         | `3s`                  | Time limit for requests made by the Consul HTTP client. A Timeout of zero means no timeout
 listen                 | `:4000`               | Accept connections at this address
+events-queue-size      | `1000`                | Size of events queue
```

Comment:
it's not sorted that much imho :) I think it's related to `workers-pool-size`. Or we should rename `workers-pool-size` to `events-queue-workers-pool-size` to fit better.